packages/venia-ui/lib/components/CartPage/ProductListing/quantity.js

```diff
@@ -94,7 +10,7 @@ const Quantity = props => {
                 quantity: props.initialValue
             }}
         >
-            <QuantityFields {...props} />
+            <QuantityStepper {...props} />
```

Comment:
@tjwiebell I had some issues implementing the re-export and alias pattern for QuantityFields in this file, since I needed to use that component here, as well as export it. This was my idea for a solution, but let me know if there is a better way to tackle this.

Reply:
* You will need to look at full file to see my approach

Reply:
Re-exporting `QuantityStepper` should make the component available for use by the default `Quantity` export. I would try this, and also go ahead and add deprecations to both of these exports.

```js
/**
 * @deprecated - moved to component directory in 12.4.0
 * @see QuantityStepper
 */
export { default as QuantityFields } from '../../QuantityStepper';
```